compiler/passes/errorHandling.cpp

```diff
@@ -546,30 +668,48 @@ bool CanThrowVisitor::enterCallExpr(CallExpr* node) {
 
 } /* end anon namespace */
 
-
 // Returns `true` if a block can exit with an error
 //  (e.g. by calling 'throw' or a throwing function,
 //   when these are not handled by try! or catch).
 // This function is useful to infer 'throws' for
 // certain compiler-introduced functions.
 
-static bool canBlockThrow(BlockStmt* block)
+static void markImplicitThrows(FnSymbol* fn, std::set<FnSymbol*>* visited, implicitThrowsReasons_t* reasons)
```

Comment:
I'd prefer to keep it and checkErrorHandling as functions since the fact that they are implemented with the various visitors seems to be to be an implementation point; where these functions are an internal API of sorts.